packages/xtemplate_test.tex

```diff
@@ -0,0 +1,78 @@
+\documentclass{article} 
+\usepackage{expl3, xtemplate, xparse, csquotes, xcolor, fontspec, xspace}
+
+\begin{document}
+\ExplSyntaxOn
+\DeclareObjectType{inlineobj}{1}
+\DeclareTemplateInterface{inlineobj}{span}{1}
+{
+  font-face: tokenlist,
+  font-shape: choice {italic, slanted, normal}=normal,
+  font-weight: choice {bold, normal}=normal,
+  font-color: tokenlist=black,
+  quote: choice {none, enquote}=none,
+}
+
+\DeclareTemplateCode{inlineobj}{span}{1}
+{
+  font-face  =  \l_font_tl,
```

Comment:
It looks like two-spaces and four-spaces indentation are mixed together throughout this file. Could you check and correct this?

Reply:
Sorry this file has been added in #51 - this PR was rebased on it as xfrac requires xtemplate. Let me open a PR fixing that test (as this one is unrelated).